ts/output/common/FontData.ts

```diff
@@ -518,6 +518,11 @@ export class FontData<C extends CharOptions, V extends VariantData<C>, D extends
    */
   public params: FontParameters;
 
+  /**
+   * Factor to multiply italic correction by for delta computations for munderover
```

Comment:
typo

Reply:
Sorry, I don't see it.  Can you point it out?

Reply:
I'm going to merge this for now, and will fix the typo later (since other approved branches depend on this and I'd like to merge them).

Reply:
I'll also change to "Factor by which to multiply italic correction for delta computations for munderover".